feature-group-definitions/margin-trim.yml

```
@@ -0,0 +1,12 @@
+name: margin-trim
+description: The `margin-trim` CSS property controls margins at the start and end of a container, removing the margins of child elements when they meet the edges of the container.
```

style, for brevity: what do you think?

```suggestion
description: The `margin-trim` CSS property removes the margins of child elements when they meet the starting or ending edges of the container.
```

---

"starting or ending edges" feels a bit abstract to me. In the typical use case it will be the top or bottom. Maybe simplify this even further?

```suggestion
description: The `margin-trim` CSS property removes the margins of child elements when they meet the edges of the container.
```

---

That works for me. 👍